aws_xray_sdk/core/recorder.py

```diff
@@ -77,7 +80,7 @@ def __init__(self):
         if type(self.sampler).__name__ == 'DefaultSampler':
             self.sampler.load_settings(DaemonConfig(), self.context)
 
-    def configure(self, sampling=None, plugins=None,
+    def configure(self, enabled=None, sampling=None, plugins=None,
```

Comment:
This seems a conflict with the `SDKConfig`. `SDKConfig` should sit on top of recorder config rather than the other way around. If users change `SDKConfig` it should automatically apply to all the key paths including recorder component.

Reply:
This would then mean that if customers want to hardcode disable their SDK, they would need to do it only from the SDKConfig module. Is this a favorable API? This may be slightly confusing to customers if they think that doing recorder.configure(enabled=False) would mean that other modules such as the LambdaContext would be disabled. What are your thoughts on this?

Reply:
`SDKConfig` is already a public module which users can import and set values. Those operations right now already take effect regardless of calling `configure` or not, based on the current implementation. So practically it is a global configuration object.

Reply:
Good point. Added the changes into the new commit